mkdocs/docs/configuration.md

```diff
@@ -268,19 +268,15 @@ catalog:
 catalog:
   default:
     uri: thrift://localhost:9083
+    hive.hive2-compatible: true
+    hive.kerberos-authorization: true
     s3.endpoint: http://localhost:9000
-    s3.access-key-id: admin
-    s3.secret-access-key: password
 ```
 
-When using Hive 2.x, make sure to set the compatibility flag:
-
-```yaml
-catalog:
-  default:
-...
-    hive.hive2-compatible: true
-```
```

Comment:
nit: leave this as is, since this is only required for hive2.

and maybe add similar block for `hive.kerberos-authorization` since that's also optional